### PR TITLE
Add devpi-client and its dependencies

### PR DESCRIPTION
**Pull request by nlewo**

### CODE REVIEW

**Comment by cillianderoiste** [comment]
Looks good to me @nlewo, except I noticed you didn't add yourself as the maintainer of devpi-client, was that intentional?

File: pkgs/development/tools/devpi-client/default.nix [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  version = "2.3.2";
+
+  src = fetchurl {
+    url = "https://pypi.python.org/packages/source/d/devpi-client/devpi-client-${version}.tar.gz";
+    md5= "bfc8cd768f983fd0585c347bca00c8bb";
+  };
+
+  buildInputs = [ pythonPackages.tox pythonPackages.check-manifest pythonPackages.devpi-common pythonPackages.pkginfo ];
+  meta = {
+    homepage = "http://doc.devpi.net";
```
Comment (pSub) [inline pkgs/development/tools/devpi-client/default.nix]: Nitpick: You don't need quotes around URLs.

Comment (nlewo) [inline pkgs/development/tools/devpi-client/default.nix]: Fixed. Thanks.

**Comment by nlewo** [comment]
@cillianderoiste It was not intentional, so I added myself in maintainers.

**Pull request merged**